### ISSUE TITLE
galaxy publication

### ISSUE THREAD

**Issue by kostyrev** [bdclark/ansible-nomad#3]
Hi Brian!
It would be great if you publish this role to galaxy!
Unfortunately, there's no any good role with sysvinit support.

**Comment by bdclark** [comment]
@kostyrevaa I will definitely consider it!  I'm not using this role in production yet (I wrote it as a proof-of-concept for a transition from Chef to Ansible).  Let me review it, add support/checksums for Nomad 0.5.0, etc. and I'll try to get it published in the next few days.

**Comment by kostyrev** [comment]
About those checksums.
I've got a suggestion:
forcing users to provide checksums is not very friendly.
Please consider using this [approach](https://github.com/kostyrevaa/kostyrevaa.consul-template/blob/master/tasks/install.yml#L22).
That way you only need to provide the desired version.

**Comment by bdclark** [comment]
I agree that forcing users to supply a checksum is not friendly.  The way it's currently implemented, users should only need to supply one when the version being referenced is not in the `nomad_checksums` dict (which is out-of-date since 0.5.0 just got released).  That said, I like the approach you referenced above and will likely implement something like it myself!
